jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -469,7 +471,7 @@ class EndpointK @JvmOverloads constructor(
                 // holding a lock inside the SctpSocket which can cause a deadlock
                 // if two endpoints are trying to send datachannel messages to one
                 // another (with stats broadcasting it can happen often)
-                TaskPools.IO_POOL.execute { dataChannelHandler.consume(PacketInfo(dataChannelPacket)) }
+                TaskPools.IO_POOL.execute { dataChannelHandler.processPacket(PacketInfo(dataChannelPacket)) }
```

Comment:
Why did this change from `consume` to `processPacket`?

Reply:
`consume` is the protected method defined in `ConsumerNode`, it's not meant to be called directly.  Previously the code had changed the visibility and exposed it as public, I fixed it back to the top-level method on Node because calling `consume` directly prevents some of the stats from being calculated correctly.